src/contour/CMakeLists.txt

```diff
@@ -173,6 +173,8 @@ if(NOT(CPACK_GENERATOR))
     endif()
 endif()
 
+set(CMAKE_INSTALL_DEFAULT_COMPONENT_NAME "contour")
```

Comment:
There are **some** `install()` calls deep in the bowels of Contour's CMake which install things into the `<Unspecified>` component... This basically makes it so that any `install()` call without a `COMPONENT xyz` argument implicitly expands to `install(... COMPONENT contour)`. Which, ridiculous as it is, [**crashes `CPack`**](http://github.com/whisperity/contour/runs/5795014486?check_suite_focus=true#step:12:52)!

![Screenshot_20220402_132001](https://user-images.githubusercontent.com/1969470/161380804-680ed94f-219e-4d90-abee-017285aef7e2.png)